### PR TITLE
Deps: add prettier config & adapt eslint config

### PR DESCRIPTION
**Pull request by Yedidyar**
what is the motivation:

https://www.youtube.com/watch?v=Cd-gBxzcsdA

if this or accepted we should add format to git hooks

add run yarn format on all project

### CODE REVIEW

**Comment by baruchiro** [comment]
Thank you!

Even before it is accepted, I'm expecting you to test it is not conflicted with ESLint.

Did you run it and run `yarn lint` afterwards?

Also, I can't see how it helps. No one will change the whole source code.

Instead, I'm asking you to add it with `lint-staged` to incrementally format every file that is touched by someone.

https://bscstudent.netlify.app/ci-checklist/

**Review by baruchiro — APPROVED** [review]
Very good work!

I will test it locally and then I will merge it.

What will be your next task? Contact me

**Comment by Yedidyar** [comment]
> Very good work!
> 
> I will test it locally and then I will merge it.
> 
> What will be your next task? Contact me

I see that the ci dont work because of the node version what should we do?

**Comment by baruchiro** [comment]
@Yedidyar we have a task for upgrading the nodeJS version: https://github.com/brafdlog/caspion/issues/464

But it is not a quick task.

You should check the `lint-staged` and find its version that matches the nodeJS v14.

**Comment by Yedidyar** [comment]
> @Yedidyar we have a task for upgrading the nodeJS version: #464
> 
> But it is not a quick task.
> 
> You should check the `lint-staged` and find its version that matches the nodeJS v14.

[cf4e936](https://github.com/brafdlog/caspion/pull/497/commits/cf4e9365c756ffda64ed2091c3a5fa91a6208129)

File: package.json [hunk 1]
```diff
@@ -51,6 +49,7 @@
     "googleapis": "^87.0.0",
     "israeli-bank-scrapers-core": "3.10.3",
     "keytar": "^7.4.0",
+    "lint-staged": "^13.3.0",
```
Comment (baruchiro) [inline package.json:52]: Version `13.3.0` is actually require node 14, as you can see [here](https://github.com/okonet/lint-staged/blob/v13.3.0/package.json#L17), and we need to take one version down.
```suggestion
    "lint-staged": "~13.2.3",
```

**Review by baruchiro — CHANGES_REQUESTED** [review]
There is still a mismatch between `prettier` and `ESLint`.
When I'm running `yarn prettier --write .` and then `yarn lint`, I see multiple warnings like this:

> ⚠  http://eslint.org/docs/rules/object-curly-newline  Expected a line break before this closing brace src\Store.test.tsx:3:56

I'm navigating to this file (`src/Store.test.tsx:3:56`), formatting it with `prettier`, and it shows me an `ESLint` worning. I'm fixing the `ESLint` warning, and formatting it again, and the warning cames back.

**Comment by Yedidyar** [comment]
@baruchiro [26c4866](https://github.com/brafdlog/caspion/pull/497/commits/26c4866a902047b7e0454ad23a65eae5ece92b65) see this commit a think it the right approach

**Comment by Yedidyar** [comment]
> Very good work!
> 
> I will test it locally and then I will merge it.
> 
> What will be your next task? Contact me

I think the next step is to move to using Vite.
what is the specific current version of node?

**Review by baruchiro — APPROVED** [review]

**Pull request merged**